### PR TITLE
Update the French and Japanese translations

### PR DESCRIPTION
**Pull request by jeandeaual**
I added the French and Japanese translations for the following items:
* shareDialog
* shareClipboard
* copyButton
* shareButton
* closeButton

I also made some minor fixes (fix the title case in French, remove unnecessary spaces and fix the Japanese exclamation marks).

### CODE REVIEW

File: src/locales/fr/translation.json [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
   "clearDataTitle": "Es-tu sûr de vouloir effacer toutes les données ?",
   "clearDataWarning": "Cela ne peut être annulé !",
-  "shareDialog": "Share this page link!",
-  "shareClipboard": "Or copy this link:",
-  "copyButton": "Copy!",
-  "shareButton": "Share!",
-  "closeButton": "Close"
+  "shareDialog": "Partage le lien de cette page !",
+  "shareClipboard": "Ou copie ce lien:",
+  "copyButton": "Copie !",
```
Comment (Noelierx) [inline src/locales/fr/translation.json:29]:
```suggestion
  "copyButton": "Copier !",
```

File: src/locales/fr/translation.json [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+  "shareButton": "Partage !",
```
Comment (Noelierx) [inline src/locales/fr/translation.json:30]:
```suggestion
  "shareButton": "Partager !",
```

**Comment by franckentien** [comment]
ok for me for French transaltion

**Comment by jeandeaual** [comment]
I just squashed several commits, there's no content change since the last review.

**Comment by mtaylor76** [comment]
Thanks for this!

**Pull request merged**